scripts/html-friendly-to-title.cjs

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  /**
+   * Converts a brand title in HTML/SVG friendly format into a brand title (as
+   * it is seen in simple-icons.json)
+   * @param {String} htmlFriendlyTitle The title to convert
+   */
+  htmlFriendlyToTitle: (htmlFriendlyTitle) =>
```

Comment:
This has to be moved to a separate file and written in cjs because `.svglintrc.js` imports it and it does not allow for esm config files.

Reply:
Yes. @ericcornelissen is working on it: https://github.com/birjolaxew/svglint/pull/41.